### ISSUE TITLE
Use 12 hour format on all cases

### ISSUE THREAD

**Issue by jpserra** [azza-bazoo/prettycron#12]
For me it's kind of inconsistent to have `toString`returning a 24h format and `getNext`returning 12h format. I think it would be better to stick with only the more universal AM/PM format.

**Comment by pekeler** [comment]
+1 on consistency
-1 on defaulting to AM/PM

**Comment by jpserra** [comment]
@pekeler true, at the time and because of personal need, I came up with this for consistency sake, but better approach would be to make it configurable
